app/src/main/java/com/hedvig/app/feature/documents/DocumentItems.kt

```diff
@@ -0,0 +1,28 @@
+package com.hedvig.app.feature.documents
+
+import android.content.Context
+import androidx.annotation.StringRes
+
+sealed class DocumentItems {
+
+    data class Header(@StringRes val stringRes: Int) : DocumentItems()
```

Comment:
Here's an interesting dilemma: making this `data class` take a `@StringRes` gives it a tight coupling with the android view-layer 🤔 

I'm not sure that I care enough to think that we should change that, but it's something to think about for sure.

Reply:
Yeah I think it makes more sense though to pass the stringRes than to inject a context in the viewModel creating the items.